cypress/full/buttons_page_spec.ts

```diff
@@ -0,0 +1,201 @@
+import { ButtonsPo } from '../support/buttons.po';
+
+describe('Buttons page test suite', () => {
+  const buttons = new ButtonsPo();
+
+  beforeEach(() => buttons.navigateTo());
+
+  describe('Checkbox', () => {
+    const checkboxDemo = buttons.exampleDemosArr.checkbox;
+
+    it('example contains 3 bttns and output window, as default - picked Middle btn', () => {
+      buttons.isButtonGroupVisible(checkboxDemo, 'Left');
+      buttons.isButtonGroupVisible(checkboxDemo, 'Middle');
+      buttons.isButtonGroupVisible(checkboxDemo, 'Right');
+      buttons.customCheckboxContains(checkboxDemo, 1, `true`);
+      buttons.customCheckboxContains(checkboxDemo, 0, `false`);
+      buttons.customCheckboxContains(checkboxDemo, 2, `false`);
+    });
+
+    it('when user clicks on the Left button it can be active or inactive', () => {
+      buttons.isButtonGroupVisible(checkboxDemo, 'Left');
+      buttons.customCheckboxContains(checkboxDemo, 0, `false`);
+      buttons.clickByText(checkboxDemo, 'Left');
+      buttons.customCheckboxContains(checkboxDemo, 0, `true`);
+    });
+
+    it('when user clicks on the Middle button it can be active or inactive', () => {
+      buttons.isButtonGroupVisible(checkboxDemo, 'Middle');
+      buttons.customCheckboxContains(checkboxDemo, 1, `true`);
+      buttons.clickByText(checkboxDemo, 'Middle');
+      buttons.customCheckboxContains(checkboxDemo, 1, `false`);
+    });
+
+    it('when user clicks on the Right button it can be active or inactive', () => {
```

Comment:
when user clicks on the Left and Right buttons they become active and all buttons in inactive state

Reply:
fixed